lib/presentation/widget/myspace/my_space_viewmodel.dart

```diff
@@ -405,9 +411,24 @@ class MySpaceViewModel extends BaseViewModel {
     };
   }
 
-  OnlineThunkAction _getAllDocumentAction() {
+  ThunkAction<AppState> _getSorter(OrderScreen orderScreen) {
+    final sorter = Sorter(orderScreen, OrderBy.modificationDate, OrderType.descending);
+    return (Store<AppState> store) async {
+      await _getSorterInteractor.execute(orderScreen).then((result) => result.fold(
+              (failure) {
+                store.dispatch(MySpaceGetSorterAction(sorter));
+              }, (success) {
+                store.dispatch(MySpaceGetSorterAction(success is GetSorterSuccess ? success.sorter : sorter));
+              }));
+    };
+  }
+
+  OnlineThunkAction _getAllDocumentAction(bool refresh) {
```

Comment:
it is not clean to me when you are using magic parameter

Reply:
I found it unnecessary in this case